### PR TITLE
[ch2223] AL CoAl Order display change to by privilege first

### PR DESCRIPTION
**Pull request by cacahouwete**
- change order by adding privilege order first
- upgrade makefile to use dorkerize for waiting mysql up

https://app.clubhouse.io/enmarche/story/2223/admin-territoires-comit%C3%A9s-bug-d-affichage-de-l-ordre-des-al-puis-co-al

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -67,7 +68,8 @@ rabbitmq-fabric:
 
 db:             ## Reset the database and load fixtures
 db: vendor
-	$(RUN) php -r "for(;;){if(@fsockopen('db',3306)){break;}}" # Wait for MySQL
+#	$(RUN) php -r "for(;;){if(@fsockopen('db',3306)){break;}}" # Wait for MySQL
```
Comment (thomasbisignani) [inline Makefile]: Can you remove this comment ?

File: templates/admin/committee/list_hosts.html.twig [hunk 2]
```diff
@@ -1,19 +1,36 @@
 {% extends 'SonataAdminBundle:CRUD:base_list_field.html.twig' %}
 
+{% import _self as list_host %}
+
+{% macro dislay_host_line(host) %}
+    <li>
+        <a href="{{ path('admin_app_adherent_show', {'id': host.id}) }}">
```
Comment (thomasbisignani) [inline templates/admin/committee/list_hosts.html.twig]: CS : please remove simple quotes for array keys  and add white spaces around arrays : 
`<a href="{{ path('admin_app_adherent_show', { id: host.id }) }}">`

Comment (cacahouwete) [reply templates/admin/committee/list_hosts.html.twig]: I always forgot it :'(

**Comment by B-Galati** [comment]
@cacahouwete Don't forget to squash :-)

**Pull request merged**